tests/specs/smoke.ts

```diff
@@ -218,6 +218,14 @@ const files = {
 
 	'file.txt': 'hello',
 
+	'import-typescript-parent.js': sourcemap.tag`
+	import './import-typescript-child.js';
```

Comment:
Can you move this inside `import-from-js.js`?

Reply:
I can, though that will require adding `--tsconfig tsconfig/tsconfig-allowJs.json` to that test's arguments, at which point there wouldn't be any tests that cover the case where TS resolution rules _aren't_ followed. If that's good with you, I'll do the move and delete the `allowJs in tsconfig.json` test.

Reply:
Ah good point, let's keep it this way then. Thanks for catching it!